File: flink/src/main/scala/org/danner/bigdata/flink/bean/misc.scala
package org.danner.bigdata.flink.bean

import java.text.SimpleDateFormat

object misc {

    case class Access(time:Long,domain:String,traffics:Long)

    case class WC(word:String,count:Long)

    case class PlatformStat(platform:String,cnt:Long,day:String)

    def getLongTime(time: String): Long = {
        var sdf = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss")
        sdf.parse(time).getTime
    }
}

File: hadoop/src/main/java/com/danner/bigdata/utils/Misc.java
package com.danner.bigdata.utils;

import java.io.Closeable;
import java.io.IOException;

public class Misc {

    public static void closed(Closeable closeable){
        if (null != closeable){
            try {
                closeable.close();
            } catch (IOException e) {
                e.printStackTrace();
            }
        }
    }
}
